### PR TITLE
SDK-899: extend the TaskSetInterface

### PR DESCRIPTION
**Pull request by sergeyspryker**
https://spryker.atlassian.net/browse/SDK-899

### CODE REVIEW

**Review by vol4onok — APPROVED** [review]

File: src/Entity/TaskSetInterface.php [hunk 1]
```diff
@@ -7,12 +7,60 @@
...
+     *
+     * @api
+     *
+     * @return array<string, bool> Key is sub-task id.
+     */
+    public function getStopOnErrorMap(): array;
+
+    /**
+     * Specification:
+     * - Overrides sub-tasks placeholders.
```
Comment (pavelmaksimov25) [inline src/Entity/TaskSetInterface.php]: It's just a getter. Why it overrides something?

File: src/Entity/TaskSetInterface.php [hunk 2]
```diff
@@ -7,12 +7,60 @@
...
+     *
      * @param array<string> $tags
      *
      * @return array<\SprykerSdk\SdkContracts\Entity\TaskInterface|string>
      */
     public function getSubTasks(array $tags = []): array;
+
+    /**
+     * Specification:
+     * - Overrides tags of sub-tasks command property.
```
Comment (pavelmaksimov25) [inline src/Entity/TaskSetInterface.php]: It's just a getter. why doesn't this method just return a value but do something else? The same  for the rest of the methods with `makes` or `overrides` words in the specification

**Review by pavelmaksimov25 — CHANGES_REQUESTED** [review]
Please merge latest master branch changes

File: src/Entity/TaskSetInterface.php [hunk 3]
```diff
@@ -7,12 +7,60 @@
...
 namespace SprykerSdk\SdkContracts\Entity;
 
+/**
+ * Defines the task set that can be used as a single task.
+ */
 interface TaskSetInterface extends TaskInterface
 {
     /**
+     * Specification:
+     * - Gets the list of task set tasks.
```
Comment (pavelmaksimov25) [inline src/Entity/TaskSetInterface.php]: `Gets` => `Returns`. The same for the rest.

**Review by pavelmaksimov25 — APPROVED** [review]

**Review by pavelmaksimov25 — APPROVED** [review]

**Pull request merged**